experimental/PropertyDDS/packages/property-dds/package.json

```diff
@@ -59,7 +59,7 @@
 		"@fluidframework/container-definitions": "workspace:~",
 		"@fluidframework/core-interfaces": "workspace:~",
 		"@fluidframework/datastore-definitions": "workspace:~",
-		"@fluidframework/protocol-definitions": "^3.2.0",
+		"@fluidframework/driver-definitions": "workspace:~",
```

Comment:
driver-definitions can be removed from devDependencies now. Based on my review of the lockfile, I think that will be true for a few other packages.

Reply:
Good catch.  There were four packages that had both normal and dev dependencies.  (Fixed).